tools/slintpad/src/properties_widget.ts

```diff
@@ -61,8 +57,13 @@ export class PropertiesWidget extends Widget {
         super.dispose();
     }
 
-    position_changed(uri: LspURI, version: number, position: LspPosition) {
-        lsp_commands.queryProperties(uri, position)
+    position_changed(uri: string, version: number, position: LspPosition) {
```

Comment:
It's too bad to lose the type here.  Should it not be `LspURI?` ?  Or why is uri in `VersionedDocumentAndPosition` not a LspURI?